RingPublishingTracking/src/main/java/com/ringpublishing/tracking/data/RingPublishingTrackingConfiguration.kt

```diff
@@ -23,6 +23,6 @@ data class RingPublishingTrackingConfiguration(
     val apiKey: String,
     val apiUrl: URL? = null,
     val applicationRootPath: String,
-    val applicationDefaultStructurePath: String,
+    val applicationDefaultStructurePath: List<String>,
```

Comment:
to zmienia interfejs, już zrobiliśmy String to może zostawmy jak jest... nie wiem

Reply:
dobra to powinno być listą od początku - przegapione + w jednym miejsce (reference guide) jest do poprawy

Reply:
W dokumentacji jest lista. Gdzieś przegapiłem po drodze zmianę, dlatego tu zmieniłem na listę tak jak ma być.